monkey/tests/unit_tests/monkey_island/cc/repositories/test_mongo_node_repository.py

```diff
@@ -147,7 +147,7 @@ def test_upsert_communication__update_existing_connection(node_repository):
 def test_upsert_communication__update_existing_node_add_connection(node_repository):
     src_machine_id = 2
     dst_machine_id = 5
-    expected_node = NODES[1].copy(deep=True)
+    expected_node = NODES[1].model_copy(deep=True)
```

Comment:
Consider adding `copy()` and `deep_copy()` to `monkeytypes.InfectionMonkeyBaseModel()`